build.gradle

```diff
@@ -27,8 +27,11 @@ allprojects {
   repositories {
     google()
     mavenCentral()
-    // Required for publishing, Dokka needs org.jetbrains.kotlinx:kotlinx-html-jvm:0.7.2
-    jcenter()
```

Comment:
yay

Reply:
Can be removed from `gradle-plugin/build.gradle` too I hope?

Reply:
I accidentally committed that. Gosh, Mondays... That'd be a workaround, but I'll keep `jcenter` for now. At least there's a guarantee until Feb 2022, whereas with the Jetbrains repo there are no guarantees. I hope they'll push all artifacts on maven central.